packages/components/src/QualityBar/QualityRatioBar.component.js

```diff
@@ -35,98 +32,114 @@ export const QualityType = {
 	NA: 'na',
 };
 
-export function QualityInvalidLine({ dataFeature, onClick, percentage, t, value }) {
+function QualityRatioBar({ onClick, type, getDataFeature, ...props }) {
```

Comment:
Added this wrapper to manage some props on a lower level (onClick, dataFeature, and dynamic class name used for styling)